.github/workflows/gh-ci.yaml

```diff
@@ -79,16 +79,25 @@ jobs:
       run: |
         python --version
         python -m pip install .  --no-deps 
+
     - name: Python information
       run: |
         which python
         which pip
         pip list
         conda info
         conda list
+
     - name: Run tests
       run: |
-        pytest -v --color=yes propkatraj/tests/
+        pytest -n 2 --cov=propkatraj --cov-report=xml -v --color=yes propkatraj/tests/
+
+    - name: codecov
```

Comment:
If you want to use a toke, use a line such as https://github.com/alchemistry/alchemlyb/blob/045f85d5019d146eb1e7c14ba817e8e4ee1adfa3/.github/workflows/ci.yaml#L73

But let's use it in "we trust Github mode for right now" (just as @IAlibay said)